miriway

```diff
@@ -1,6 +1,22 @@
 #! /bin/sh
 set -e
 
+### Gtk Theming Tweaks
+config="${XDG_CONFIG_HOME:-$HOME/.config}/gtk-3.0/settings.ini"
+if [ ! -f "$config" ]; then exit 1; fi
```

Comment:
You mean to delete the complete if block highlighted ??

Reply:
I mean...
```bash
if [ -f "$config" ]; then
  // parse file
  // call gsettings
fi
```

Reply:
Oh, ok implementing it....

Reply:
> I mean...
> 
> ```shell
> if [ -f "$config" ]; then
>   // parse file
>   // call gsettings
> fi
> ```

The problem with your suggestion is it may be doing the work but with the following error :-
xyz: line 6: //: Is a directory
xyz: line 7: //: Is a directory

Is there any suggestion to get rid of it or is this Normal ?

Reply:
I recommend removing the suggestion, i am removing it as i feel its problematic , you can suggest a better one ??

Reply:
The suggestion was not meant literally "// parse file" was shorthand for your code parsing the file. Similarly, "// call gsettings" was short for your code calling gsettings.

Reply:
So should i keep it, i think removing the section will be better, because i dont think its useful

Reply:
@AlanGriffiths  sorry for the misunderstanding, i have implemented your suggestion, took me a very long time to understand, hope now its fine ?/